com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -836,6 +849,15 @@ public bool supportsHDR
             set { m_SupportsHDR = value; }
         }
 
+        /// <summary>
+        /// Graphics format requested for HDR color buffers.
+        /// </summary>
+        public HDRColorBufferPrecision hdrColorBufferPrecision
+        {
+            get { return m_HDRColorBufferPrecision; }
+            set { m_HDRColorBufferPrecision = value; }
+        }
```

Comment:
```suggestion
        public HDRColorBufferPrecision hdrColorBufferPrecision
        {
            get => m_HDRColorBufferPrecision;
            set => m_HDRColorBufferPrecision = value; 
        }
```

you can write this

Reply:
While I agree it's nicer to read, all the other getters are in above style and it's the same amount of chars except for the return.
If we want to change this, I think we should change them all in a batch.